modules/op-release-notes-1-8.adoc

```diff
@@ -123,13 +123,30 @@ Because this feature is available by default, you no longer need to set the `pip
 === Operator
 
 // [Hub]
-* With this update, as an administrator, you can configure your local {tekton-hub} instance to use a custom database rather than the default database.
+* With this update, as an administrator, you can configure your local {tekton-hub} instance to use a custom PostgreSQL 13 database rather than the default database. To do so, you create a `tekton-hub-db` `Secret` instance. For example:
```

Comment:
Not a big deal, but since you've already directly addressed the user, couldn't you simply say, `To do so, create a `tekton-hub-db` `Secret` instance. For example:`

Reply:
Updated. Thanks.